src/styleeditorwindow.cpp

```diff
@@ -9,16 +9,15 @@
  * Initializes the window components and configures the StyleEditorWindow
  */
 StyleEditorWindow::StyleEditorWindow(QWidget *parent)
-    : QWidget(parent),
+    : QDialog(parent),
       m_ui(new Ui::StyleEditorWindow),
       m_currentlyClickedButton(Q_NULLPTR),
       m_currentSelectedFontButton(Q_NULLPTR),
       m_currentSelectedThemeButton(Q_NULLPTR),
       m_isFullWidthClicked(false)
 {
     m_ui->setupUi(this);
```

Comment:
We may also want to change `class="QWidget"` to `class="QDialog"` in `styleeditorwindow.ui`. I think that change can only be made using a text editor, though. Same for the other two classes.